Integrations/integration-Salesforce.yml

```diff
@@ -4,9 +4,20 @@ commonfields:
524WSnXvEGXeoVw64S2aLG/NwWzZ4OWzFg/MMTH8Ftbc1UK58SGxZ+hsCMwAcKIMDCwRVfvfqRHtzhkhBMM5KO3bGmPdt8ViUyXMvZcyd5diTgbC5DU3mzd+2OTBnxXFuUqbB+XDxPA8ojlViTWuzvSOXITyJ90VCABAtjD0FpkFWxRzbm6FgRbosVjWgh50k3uxWLBfTUv1awifP848T2Huoc7qb2/8BJOZtTsDG7b9iXnNlVb6Sq3gScK8R8Vys0dWnSuzpdqLp6LpBACArliQyvYqagR7FQPmWM8ZNSfOrd4WEyNoyXrj79Dl8ZK02emLD0od3VteJhftD1DVvoEa3gOy3nyMMO9genB8x9jEwTB/bwYnVw5hzjFli8n6txVN/D+rY3ziHM+W9xHDi2TTXSPp3uelYvMdQnnknCluOAEAINszfDnWgxqFLHruaPhp2QAbvsnPE9DvU1iXwbcOLhN76q+RyrWPCGX7RsL8+xQ+88XwnshZ0YFcJTo8cctnjHv0pg9Nr1k+ixVuKzH5bN/kLaD0s+FhWmklAdnfSZi7Wda97/DSmFysvz9z++jFfJTBNK2kAgCAZz2y6hWoEeyDLHpuZM80zZ7DkJbtD6JKO/tHD4151vqtjWdKun0p0ZvvxFrjHwnzrsPMtbDuPJS8uoS5jxPD/UXy+mvMPB5sD1AzOEjN8GDy/iA2/P0y857GzH+SMH8nNdw21Zo1Yrg/lE1Pk5n/dUlrfDxbHr5c7rUvQckPB5S3O2ZsUzYAQBbtetC5OEe2Wa99PJZYs0qT2zkndMNnIF6SBvTeTS/IGPZLhN6hN0ha/R1ysX6lWK59GpcaX5bLzvX8iqbEnG5Jc1dJ5WZXNsnAxaKdl0r25yXD+RhOFrXKZfutlDUvy/a4FxDD7kxvD/XHi2d4uzYAAPy0XSrXbqKmP9tZNEx56xuPsRXtToLmJ9Jge7LwQNrfvzgtP/Bs95ZRCXUlgZbVz1xmumej2zafi4rhuSipHfPrmMl/PwN1DytpRq4NLuXBOA36aNYDMgAAmf9Yv1xizR/yzRaQRc/eNmtqhoew2tQzNw8un+YSwqLJ4MsFgDkPDHAqjL6HmOHjiglzOmZcw0vb9wnzH8n2jr4tzWKPAgAAkB8UpVKtm1Ta+2cuiwZzuhN38psWVvALPoclLTUAAAD8PrwDTGDeQ0rf2OEZ2kgOWgEvbRzBrFnNlB69GOq+AAD8wVLH4tLoFcT0dyqVGWkBh1sbd4wfkZn7cLYw9HaU7+9AAAAAf5BbHpREbeTrdM22pxUTBilNl5T/htI3lt55xqr9yXQI1nMBAAAgaIPnSVrzLrpmx0EK0+6mRz5kyAyewFrzq1gbXIaOCwAAgBgtWto7+mcyC/83t3b7EQqHhlNqJ79SZwT7+UzvbM+mCxAAAMAJsaJ/caa3/lfEDJ3cmscmB+84EGCn4EodNUMenO+WCo1XndyhIAAAwOf6BYGNXkXMqMUnrCk6BOnnG5wVq3UYa837hFXDb0y79U4aAACAoi1ni7VPETN4PHfHVj6vA4L0yc5TrvJ1UGFAy+5bYQocAABTw7cHaaZUvxabwc40SBuQSZ9wcK7wWcrBLrlcvy6ddQEAADBV8DGTklr7PDX9rekWccODIH28ZY1Ki1+n2y6VmjcIt9XPRQAAAFMNSZbVUq25krDQU1Y/FsP0uz9u7rc7BK1gq1hqfEPUnLPQdAEAAKB0Dytyyb6GGsEY39bBN05DW/jvyRKtMD0QJMx3pGL9i6h7EILzDAAAgDa4DJdqK6nubU6C0CFe8oDDw0mZO5k1Rwdk3f9ZtnfkYyi/kaAZAwAAIB8vkYrDl0q604XN4DGlMs6XjJ7y+wT5fkdiRWOS6t7cURh6OYpnaeYyAAAwXEnQ7PNkzf4sYcGWdEFppXWKtm630pnO1PBH5FL9GuH6kXPQbAMAAJDTIirq9hWU+T+hVnQwxw8Q+aZwbaEveU00w5g38VDTP4SZ+xOpVPswyg/IaC4BAABsCBeLjddizf0SscJHKA/QSeBKr5lpzoLbvq0wL/33EdPnczV+KhXtL4pq/TVzNzgDAABogzRbGnq7qNk9xAxadE0SpO8Y5weI3Hm/2JUmr7nk30OT3xJIpR1iw+nCpdG/QPl1BAEAAMyHA0RUGHiRrI5+AJvhHdQMfpW7a2ec69s6MQdZd+ZlYFb62rFy544YG/5uiXl9WHPeK/YMnY0GoG0bAID5RmFdhmr15RJzP4nN4B7FCncra3fE/DBxXgxe0uyJO81JeztNvm7C/D2y6d+TVe2PZgr15ajf7kAAAADzmvu3i0SNLiTlxocwcy3ZcPfm7tzOSwWTNWp7DgVrO6b8azH9uHPt9lhZuzPGhrdLMNw75OTrz/Ruugj1bxAQgutzAAAsGPKnoU8MZIVi/VyhVFtBNceUteYO0jce53hWXRmLFWM2yx9O+sMit3prrKzZEVM94FfmBjFr3pgx7Heh3vXno3VBZsHPbwYAAG58LOt5+Gyh59E3CcXGl2Q9+G9i+Lt4KUFZ+3isVFtpBjvdm1wo8xL9ib8vWUpAk1dZdz1R9wy5bH9Q7t18CTLsTrQyXooAAABOta0tqLhL7uRZdWHL65eqjatlvakSrWkT3d2LmZ/Wf3kppNOKeIY7BQPzo+T/ty1W+vgVuSAmWnOvrDUa/NZJGpTLo69At4yega49mi0DAABAVyLPrBOJuNo5S1g18maxZ/SfiGaXJKP5I5m5Taw5T1DdPcIDLZcyf1L+3j1GP3XycxPZuO4eJqqzG+vNBladB4SSXRDV+jU4+XuE8sg56PsuTg/9eCnmjwEAAACsWIy0eClaF2eQ6WKUZNlo1aOvWNK96a1LCyN/ny3bN4rluimWGv8pq86DiY/Iqr1FKDXWZ4v1HyWv382qrtWh2XmBd/dZm9/Y0bPhlei2zeei/gEZ5QeyaOXg0gW5dgoAAAAAAOD/AJTdWvpkIYlkAAAAAElFTkSuQmCC
```

Comment:
@yaakovi did you get the image from Eynav? it seems larger. it should be around 4KB

Reply:
no, i only improved the integration so the image was already there